workshops/maze/src/final/main.js

```diff
@@ -0,0 +1,19 @@
+var maze;
+var mazeImg;
+
+function setup() {
+  createCanvas(200,200);
+
+  maze = createSprite(100, 100);
+  mazeImg = loadImage("http://i.filmot.org/7xDfW51.png");
```

Comment:
Should this use Surrogate? Surrogate works at https://surrogate.hackedu.us/i.imgur.com/7xDfW51.png.